config/rules.rb

```diff
@@ -169,9 +169,9 @@ def summarise_one_feedback(f, opp)
       # eligibile
       cohort = (COHORT_JOBS.select { |j| j[:posting_id] == Util.posting(opp) }.first || {}).dig(:cohort)
       eligibility_value = (f['fields'].select { |f|
-        f[:_text].include?('eligible') ||
+        f[:_text].include?('are they eligible') ||
         f[:_text].include?('elligible') ||
-        f[:_text].include?('cohort they can join')
```

Comment:
are you sure there isn't at least one (legacy?) feedback form with this wording?
i.e. should we keep "cohort they can join" but also add "cohort they are"?

Reply:
yes good point, will add